### PR TITLE
Adds in viewability.js

### PR DESCRIPTION
**Pull request by kahwee**

### CODE REVIEW

File: files/viewability/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "viewability",
+  "repo": "kahwee/viewability",
+  "files": {
+    "basePath": "dist/"
```
Comment (jimaek) [inline files/viewability/update.json]: Please add the include parameter and set it to include only the minified file.

**Comment by kahwee** [comment]
Tell me if this works. Thanks.

File: files/viewability/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "viewability",
+  "repo": "kahwee/viewability",
+  "files": {
+    "basePath": "dist/",
+    "include": ["viewability.js", "viewability.min.js"]
```
Comment (jimaek) [inline files/viewability/update.json]: Can you remove viewability.js? The min file is enough for production

**Comment by jimaek** [comment]
@megawac Wow the bot is going crazy

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by kahwee** [comment]
:laughing: Thanks @jimaek

**Comment by megawac** [comment]
I think the github api gave it an error or something

**Pull request closed without merging**